docs/source/tutorials/image.ipynb

```diff
@@ -451,14 +1093,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot_examples([59915])"
+    "plot_near_duplicate_issue_examples(near_duplicate_issues_df, num_examples=5)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Given label is **4** but also looks like **9**:\n"
+    "### View dark images\n",
+    "\n",
+    "In this section, we display examples from the dataset that are too dark. Sometimes, when the images are too dark, it becomes challenging for both annotators and model to assign\n",
+    "then into one class, leading to poor quality data for training and testing.\n",
+    "\n",
+    "The `dark_issues` DataFrame tells use which examples are considered to be too dark, and we can sort them via the `dark_score` which quantifies how severe this issue is for each image (lower values indicate more severe intances of a type of issue). This allows us to visualize examples in the dataset that are considered to be too dark."
```

Comment:
```suggestion
    "### Dark images\n",
    "\n",
    "Datalab can also detect low-quality images in the dataset, such as those that are abnormally dark. It can be challenging for both annotators and models to assign\n",
    "a proper class label for low-quality data, which can hamper model-training and testing.\n",
    "\n",
    "The `dark_issues` DataFrame tells use which examples are considered to be abnormally dark, and we can sort them via the `dark_score` which quantifies how severe this issue is for each image (lower values indicate more severe instances of a type of issue). This allows us to visualize images in the dataset considered to be too dark."
```